tests/utils/verification/volumeproperties.go

```diff
@@ -173,7 +173,7 @@ func VerifyDetachedStatus(name, hostName, esxName string) bool {
 	log.Printf("Confirming detached status for volume [%s]\n", name)
 
 	//TODO: Need to implement generic polling logic for better reuse
-	for attempt := 0; attempt < 30; attempt++ {
+	for attempt := 0; attempt < 60; attempt++ {
```

Comment:
Isn't it better to control const variable to adjust `attempt` until we come to #1301? I would prefer to have const so we don't need to dive into code to adjust it and do some adjustment by controlling const variable.